gcsio/src/test/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageNewIntegrationTest.java

```diff
@@ -1337,7 +1345,11 @@ private GoogleCloudStorage createGoogleCloudStorage(GoogleCloudStorageOptions op
   }
 
   private String getTestResource() {
-    return name.getMethodName() + "_" + UUID.randomUUID();
+    return getMethodName() + "_" + UUID.randomUUID();
+  }
+
+  private String getMethodName() {
+    return name.getMethodName().replace('[', '-').replace(']', '-');
```

Comment:
why is this required? Is there any behaviour change in Veneer?

Reply:
With parameterization each `Test` method will gets converted, for eg if test method name was `testMe` with parameterization it will get converted into `testMe[0]`, `testMe[1]` etc depending upon number of variations are there.

As we were using `methodName` as a objectPath for gcs. It was creating some issues in request validation due to url-encoding. To bypass this I have added additional filtering/replacements.